lib/components/src/blocks/Story.tsx

```diff
@@ -34,12 +35,16 @@ type IFrameStoryProps = CommonProps;
 
 type StoryProps = InlineStoryProps | IFrameStoryProps;
 
-const InlineStory: FunctionComponent<InlineStoryProps> = ({ storyFn, height, id }) => (
+const InlineStory: FunctionComponent<InlineStoryProps> = ({ storyFn, height, id = uuidv4() }) => (
   <Fragment>
-    {height ? <style>{`#story--${id} { min-height: ${height} }`}</style> : null}
-    <Fragment>
-      {storyFn ? createElement(storyFn) : <EmptyBlock>{MISSING_STORY(id)}</EmptyBlock>}
-    </Fragment>
+    {height ? (
+      <style>{`.sbdocs .story--${id} { min-height: ${height}; transform: translateZ(0); overflow: auto;`}</style>
```

Comment:
Why do you need to set transform ?

Reply:
I need to reset the wrapper context for dealing with `position: fixed` or `position: sticky` usage in the components.
Take a look at  #8011

Reply:
Whoa that's impressive the impact of transform over position. Thanks again ;)